src/Microsoft.DotNet.Interactive.Http.Tests/ParserTests.Combinatorial.cs

```diff
@@ -198,7 +204,22 @@ public static IEnumerable<object[]> GenerateInvalidRequests()
                 ++generationNumber;
                 yield return new object[]
                 {
-                    new HttpRequestNodeSyntaxSpec(method, url, version, headerSection, bodySection),
+                    new HttpRequestNodeSyntaxSpec(namedRequest, method, url, version, headerSection, bodySection),
+                    generationNumber
+                };
+            }
+
+            foreach (var namedRequest in InvalidNamedRequests())
+            foreach (var method in ValidMethods())
+            foreach (var url in ValidUrls())
+            foreach (var version in ValidVersions())
+            foreach (var headerSection in InvalidHeaderSections())
```

Comment:
If you look at the other examples in this method, each block of nested `foreach` loops includes only a single invalid element. You have both `InvalidNamedRequests` and `InvalidHeaderSections` in the same loop. Is that intentional?